### PR TITLE
fix CT crash on numeric filter

### PR DESCRIPTION
**Pull request by MeirShpilraien**

### CODE REVIEW

File: src/redisearch_api.h [hunk 1]
```diff
@@ -183,6 +183,8 @@ MODULE_API_FUNC(int, RediSearch_QueryNodeGetFieldMask)(RSQNode* qn);
 
 MODULE_API_FUNC(RSResultsIterator*, RediSearch_GetResultsIterator)(RSQNode* qn, RSIndex* sp);
 
+MODULE_API_FUNC(void, RediSearch_SetMaxResultsToUseCT)(size_t num);
```
Comment (mnunberg) [inline src/redisearch_api.h]: This doesn't need to be an API function; just access RSGlobalConfig directly from the tests.

Comment (MeirShpilraien) [reply src/redisearch_api.h]: This should defensively be an api function cause the user might want to change this value for performance running.

Comment (mnunberg) [reply src/redisearch_api.h]: Hrm, in that case, let's rename it to something like SetCriteriaTesterThreshold

Comment (MeirShpilraien) [reply src/redisearch_api.h]: Done

**Comment by MeirShpilraien** [comment]
Approved by @swilly22

**Pull request merged**